governance/addin-api/README.md

```diff
@@ -1,3 +1,9 @@
 # Governance Addin Api
 
-Governance Addin Api interface
+The addin API is for customising the governance structure of your DAO with plugins. 
+For example: 
+* Use [NFTs](https://docs.realms.today/DAO-Management/createing-DAOs/NFT-Community-DAO) as [governance power](https://github.com/solana-labs/governance-program-library/tree/master/programs/nft-voter)
```

Comment:
There was a typo in the docs URL which was fixed, please update accordingly 
`createing-DAOs` -> `creating-DAOs`